account_asset_management/views/account_asset_group.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_asset_group_view_form" model="ir.ui.view">
+        <field name="name">account.asset.group.form</field>
+        <field name="model">account.asset.group</field>
+        <field name="arch" type="xml">
+            <form string="Asset Group">
+                <group>
+                    <group>
+                        <field name="company_id" invisible="1" />
```

Comment:
Removed this field

Reply:
if I delete this, I get this error:

```
odoo.tools.convert.ParseError: while parsing /mnt/environment/16.0-mig-account_asset_management/account_asset_management/views/account_asset_group.xml:3
Error while validating view near:

<form string="Asset Group" __validate__="1">
                <group>
                    <group>

Field 'company_id' used in domain of field 'parent_id' ([('company_id', 'in', [company_id, False])]) is restricted to the group(s) base.group_multi_company.

View error context:
{'file': '/mnt/environment/16.0-mig-account_asset_management/account_asset_management/views/account_asset_group.xml',
 'line': 1,
 'name': 'account.asset.group.form',
 'view': ir.ui.view(965,),
 'view.model': 'account.asset.group',
 'view.parent': ir.ui.view(),
 'xmlid': 'account_asset_group_view_form'}
```

I have assumed that this was correct because of this example: 
https://github.com/OCA/OCB/blob/16.0/addons/account/views/account_account_views.xml#L92

Reply:
The problem is in the parent_id field.
```
parent_id = fields.Many2one(
        comodel_name="account.asset.group",
        string="Parent Asset Group",
        ondelete="restrict",
        check_company=True,
    )
```
Si se pone en vista que el campo company_id con restricción de grupo "base.group_multi_company". Este campo parent_id no pasara la restricciones  añadidas aquí.

https://github.com/OCA/OCB/blob/16.0/odoo/addons/base/models/ir_ui_view.py#L2974

This is why you need the duplicate field with the invisible property.

Reply:
Yeah, it makes sense. @moylop260 it seems the linter shouldn't rise on this case. I think the duplicated field in view should take into account `groups` attribute.

Reply:
I have checked the lint in odoo project itself and it is detecting the following cases: 

```txt
odoo/addons/account/views/account_move_views.xml:166 Duplicate xml field "company_id" in lines 167
odoo/addons/account/views/account_account_views.xml:92 Duplicate xml field "company_id" in lines 104
odoo/addons/account/views/account_journal_views.xml:253 Duplicate xml field "company_id" in lines 257
odoo/addons/hr/views/hr_employee_views.xml:276 Duplicate xml field "company_id" in lines 281
odoo/addons/hr_holidays/views/hr_leave_stress_day_views.xml:32 Duplicate xml field "company_id" in lines 34
odoo/addons/hr_recruitment/views/hr_recruitment_views.xml:52 Duplicate xml field "company_id" in lines 53
odoo/addons/product/views/product_pricelist_item_views.xml:60 Duplicate xml field "company_id" in lines 85
odoo/addons/project/views/project_views.xml:691 Duplicate xml field "company_id" in lines 692
odoo/addons/project/views/project_views.xml:1509 Duplicate xml field "company_id" in lines 1510
odoo/addons/resource/views/resource_views.xml:61 Duplicate xml field "company_id" in lines 64
odoo/addons/stock/views/product_strategy_views.xml:8 Duplicate xml field "company_id" in lines 37
odoo/addons/stock/views/stock_quant_views.xml:111 Duplicate xml field "company_id" in lines 120
odoo/addons/stock/views/stock_quant_views.xml:363 Duplicate xml field "company_id" in lines 387
odoo/addons/stock/views/stock_orderpoint_views.xml:43 Duplicate xml field "company_id" in lines 62
odoo/addons/stock_picking_batch/views/stock_picking_batch_views.xml:63 Duplicate xml field "company_id" in lines 76
```

So, I think it is a valid case

However, we are migrating this [pylint-odoo](https://github.com/OCA/pylint-odoo) check to [new project](https://github.com/OCA/odoo-pre-commit-hooks) and it will be deprecated in pylint itself and we are working in progress in the new project


In order to avoid delaying your contribution you even can disable this check [only for this file](https://github.com/oca/pylint-odoo#skip-one-xml-check) or you can [remove the check for this project](https://github.com/OCA/account-financial-tools/blob/f9ddfe348fa7cd54e2a7e10482b69d1da65a7164/.pylintrc-mandatory#L28)

In the next sync of configuration files I hope fix this issue in the new tool

Reply:
FYI https://github.com/OCA/odoo-pre-commit-hooks/issues/10

Reply:
Thanks for checking. Yeah, disabling it for this file can work for now. @RodrigoBM can you do it?

Reply:
Yeah, I do it